build/metadata/github-actions/action.yml

```diff
@@ -9,30 +9,36 @@ inputs:
     description: 'Key of the LaunchDarkly project associated with this repository. Found under Account Settings -> Projects in the LaunchDarkly dashboard. Cannot be combined with `projects` block in configuration file.'
     required: false
   accessToken:
-    description: 'Token with write access to LaunchDarkly project.'
+    description: 'A token with write access to the LaunchDarkly project.'
     required: true
   repoName:
-    description: 'Sets the repository name for this project. This is useful if you have a single monorepo containing multiple YAML configurations each mapping to its own projKey. Each YAML configuration must have a unique LD_PROJ_KEY and LD_REPO_NAME combination so it displays correctly in the LaunchDarkly dashboard. Defaults to the current Github repository.'
+    description: 'The repository name. Defaults to the current GitHub repository.'
+    required: false
   baseUri:
-    description: 'Set the base URL of the LaunchDarkly server for this configuration.'
+    description: 'The base URL of the LaunchDarkly server for this configuration.'
     default: 'https://app.launchdarkly.com'
+    required: false
   contextLines:
     description:
       The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.
-    default: 2
+    default: '2'
+    required: false
   allowTags:
-    description: Enables storing references for tags. The tag will be listed as a branch.
-    default: false
+    description: Enable storing references for tags. The tag will be listed as a branch.
```

Comment:
Okay that lines 27, 31, 39 don't have single-quotes around the `description` content, but lines 12, 15, 18 do? I'm not familiar with the action format and not sure how to test.

Reply:
i believe they only _need_ to be wrapped in quotes if they contain a subset of special characters or spaces? But i can wrap all in quotes for consistency

Reply:
Up to you if it's not required, I just wasn't sure. Thanks for considering!